### PR TITLE
RESTful responses using standard HTTP codes

### PR DESCRIPTION
**Pull request by lionel-panhaleux**

### CODE REVIEW

File: agency/README.md [hunk 1]
```diff
@@ -261,19 +244,16 @@ For `reason_code`, options are:
...
-* `320: Error: vehicle trip has not been properly started`
+## Responses
+
+RESTful responses are returned for all endpoints, namely:
+
+* **200:** OK, operation successful. For `PUT` operations, object existed and has been updated.
+* **201:** For `PUT` operations, new object created
+* **400:** Malformed request.
+    In this case, the answer presents the list of errors as an `application/json` body;
+    Each error in the list is a string in plain english.
```
Comment (sebdiem) [inline agency/README.md]: Could we use a more structured response here ? It could be interesting to at least provide a mapping between the expected arguments and the related error.

Comment (HenriJ) [reply agency/README.md]: I gave 2 examples of simple and clear error response body in previous issue : https://github.com/CityOfLosAngeles/mobility-data-specification/issues/127

Was closed as dupe of https://github.com/CityOfLosAngeles/mobility-data-specification/issues/105

From my experience, returning a list of errors is rarely usefull, but maybe you have an other pov ?
